.github/workflows/gh-release-notes.yml

```diff
@@ -2,9 +2,6 @@ name: "Github-ReleaseNotes"
 
```

Comment:
Why keep this workflow if we have the release please one ? just for flexibility if we want a simple release system on some repos ?